.classpath

```diff
@@ -11,7 +11,6 @@
 			<attribute name="module" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="var" path="USERLIBS_DIR/TalonSRXLibJava.jar"/>
```

Comment:
We may need to re-add this one and the other two libraries below if Eclipse cannot find them in it's standard include paths, but let's go ahead and merge this and see.